src/Reflection/MethodReflection.php

```diff
@@ -368,7 +368,7 @@ protected function extractPrefixedWhitespace($haystack, $position)
      *
      * @param array $haystack
      * @param int $position
-     * @return bool
+     * @return bool|void
```

Comment:
```suggestion
     * @return bool|null
```

Surely with void meaning no return, this should be null?

Reply:
Hmm, yeah, perhaps `bool|null` is more precise.

I went with `|void` beacause of no declared `return`

Reply:
a9d601d091c8478f64275ba15836689200c7e0f0